main.go

```diff
@@ -17,22 +17,31 @@ limitations under the License.
 package main
 
 import (
-	"flag"
-	"os"
-	"sigs.k8s.io/aws-alb-ingress-controller/controllers/ingress"
-	"sigs.k8s.io/aws-alb-ingress-controller/controllers/service"
-
+	"github.com/spf13/pflag"
 	"k8s.io/apimachinery/pkg/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"os"
+	"sigs.k8s.io/aws-alb-ingress-controller/controllers/ingress"
+	"sigs.k8s.io/aws-alb-ingress-controller/controllers/service"
+	"sigs.k8s.io/aws-alb-ingress-controller/pkg/aws"
+	"sigs.k8s.io/aws-alb-ingress-controller/pkg/aws/throttle"
+	"sigs.k8s.io/aws-alb-ingress-controller/pkg/networking"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
 
 	elbv2v1alpha1 "sigs.k8s.io/aws-alb-ingress-controller/apis/elbv2/v1alpha1"
 	elbv2controller "sigs.k8s.io/aws-alb-ingress-controller/controllers/elbv2"
 	// +kubebuilder:scaffold:imports
 )
 
+const (
+	flagMetricsAddr          = "metrics-addr"
+	flagEnableLeaderElection = "enable-leader-election"
+	flagK8sClusterName       = "k8s-cluster-name"
```

Comment:
let's keep it  as `cluster-name` to remain compatible with old controller's flags

Reply:
done